packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/FIAPRequestHandler.h

```diff
@@ -17,4 +17,20 @@ typedef void (^ProductRequestCompletion)(SKProductsResponse *_Nullable response,
 
 @end
 
+@protocol RequestHandler <NSObject>
+
+- (void)startProductRequestWithCompletionHandler:(ProductRequestCompletion)completion;
+
+@end
+
+@interface DefaultRequestHandler : NSObject <RequestHandler>
+- (instancetype)initWithRequestHandler:(FIAPRequestHandler *)handler;
+@property FIAPRequestHandler *handler;
+@end
+
+@interface TestRequestHandler : NSObject <RequestHandler>
```

Comment:
there shouldn't be a test stub in non-test module.

Reply:
moved test stubs into mocks file